docs/source/release/v6.5.0/Framework/Algorithms/New_features/34203_AddContainerScatteringToDiscus.rst

```diff
@@ -0,0 +1 @@
+- The algorithm :ref:`DiscusMultipleScatteringCorrection <algm-DiscusMultipleScatteringCorrection>` now simulates scattering in the a container (and more generally in any sample environment components) if a user sets up the shapes\materials in the input workspace prior to running the algorithm
```

Comment:
```suggestion
- The algorithm :ref:`DiscusMultipleScatteringCorrection <algm-DiscusMultipleScatteringCorrection>` now simulates scattering in the a container (and more generally in any sample environment components) if a user sets up the shapes\materials in the input workspace prior to running the algorithm.
```

Reply:
Remove 'the' or  'a' in 'in the a container'?